pints/tests/test_toy_fitzhugh_nagumo_model.py

```diff
@@ -65,19 +65,19 @@ def test_sensitivities(self):
         parameters = [0.2, 0.7, 2.8]
 
         # Test with initial point t=0 included in range
-        times = [0, 7, 12]
-        sols, sens = model.simulateS1(parameters, times)
-        print(sens)
-        print(sens[:, 0][:, 2])
-        self.assertAlmostEqual(sens[:, 0][:, 2][1], 5.01378, 5)
-        self.assertAlmostEqual(sens[:, 1][:, 1][2], 0.82883, 4)
+        sols, sens = model.simulateS1(parameters, [0, 7, 12])
+        self.assertAlmostEqual(sens[1, 0, 2], 5.01378, 5)
+        self.assertAlmostEqual(sens[2, 1, 1], 0.82883, 4)
 
         # Test without initial point in range
         sols, sens = model.simulateS1(parameters, [7, 12])
-        print(sens)
-        print(sens[:, 0][:, 2])
-        self.assertAlmostEqual(sens[:, 0][:, 2][0], 5.01378, 5)
-        self.assertAlmostEqual(sens[:, 1][:, 1][1], 0.82883, 4)
+        self.assertAlmostEqual(sens[0, 0, 2], 5.01378, 5)
```

Comment:
Have you changed the shape of sensitivities that are returned? Or is this just aother way to refer to them? If the former, then we need to check that the notebooks (F-N, Goodwin and Lotka-Volterra) still work with the gradient-based samplers (they weren't before as the sensitivities were wrong).

Reply:
This is just a simpler way to say the same thing!

Old code says:
- Take a 3d array, turn it into a 2d array, by selecting only bits where 2nd indice is 2 (first brackets)
- Now make a 1d array by selecting only bits where 2nd indice (old 3d indice) is 2
- Now select a value from the resulting 1d array

It's much easier to just immediatly select the item you want :D

Reply:
Cool -- yep, looks neater! Thanks -- feel free to merge!

Reply:
Actually, I just did it! (I didn't think I was allowed to merge.)